source/Calamari.ConsolidateCalamariPackages/Calamari.ConsolidateCalamariPackages.csproj

```diff
@@ -1,13 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
+        <RootNamespace>Octopus.Calamari.ConsolidatedPackage</RootNamespace>
+        <AssemblyName>Octopus.Calamari.ConsolidatedPackage</AssemblyName>
```

Comment:
Why is the project name different to the produced assemly?

Reply:
That is a good question :D
I'm not a dotnet developer originally, so not sure of conventions.
I felt weird having a project called Octopus.X inside the Calamari project - but figured that the published assembly should show ownership (and thus have an octopus prefix).
Happy to align the project/assembly names - should I update the project name to match the assembly, or vice-versa?

Reply:
Ok - in a separate PR I'll update the project names to match the assembly names 👍